static/usage/fab/theming/css-shadow-parts/demo.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Fab</title>
+  <link rel="stylesheet" href="../../../common.css" />
+  <script src="../../../common.js"></script>
+  <script type="module" src="https://cdn.jsdelivr.net/npm/@ionic/core@6/dist/ionic/ionic.esm.js"></script>
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@ionic/core@6/css/ionic.bundle.css" />
+
+  <style>
+    ion-fab-button::part(native) {
```

Comment:
Can we change the colors a bit? They are hard to see in dark mode: 
![image](https://user-images.githubusercontent.com/2721089/198724749-eb824753-ca97-4fc8-a99f-e096f438772e.png)

Reply:
Finally figured out how to get the hover and active BG colors working 😆 I adjusted it to match the custom properties playground.